packages/tpetra/core/src/Tpetra_FECrsGraph_def.hpp

```diff
@@ -308,6 +379,42 @@ void FECrsGraph<LocalOrdinal, GlobalOrdinal, Node>::beginFill() {
 
 }
 
+// template<class LocalOrdinal, class GlobalOrdinal, class Node>
```

Comment:
This function is added to the file, but completely commented out.  Is it needed?

Reply:
Ouch, same as above. I'll fix all this sloppy nonsense.